src/main/java/com/softserve/kickscooter/kickscootervehicle/management/service/ScooterManagementService.java

```diff
@@ -0,0 +1,70 @@
+package com.softserve.kickscooter.kickscootervehicle.management.service;
+
+
+import com.softserve.kickscooter.kickscootervehicle.management.dao.Scooter;
+import com.softserve.kickscooter.kickscootervehicle.management.dao.ScooterRepository;
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterCreateDto;
+import com.softserve.kickscooter.kickscootervehicle.management.dto.ScooterInfoDto;
+import lombok.AllArgsConstructor;
+import org.springframework.core.convert.ConversionService;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.UUID;
+
+@Service
+@AllArgsConstructor
+public class ScooterManagementService implements ManagementService {
```

Comment:
Management is a redundant word, use words that really means something.

Reply:
`Register` is definitely more specific than management. Also, avoid verbs in class names. I would call it `ScooterRegistrationService`.

Clean Code: 
> Classes and objects should have noun or noun phrase names like Customer, WikiPage, Account, and AddressParser. Avoid words like Manager, Processor, Data, or Info in the name of a class. A class name should not be a verb.